docs/api/twine.cli.rst

```diff
@@ -0,0 +1,4 @@
+twine.cli module
```

Comment:
Putting this here as an "API" means it is supported for people to be calling into and we definitely don't want that. The CLI module should be able to change as we need it to.

Reply:
Maybe I shouldn't used have used "API". My intended audience for this documentation isn't users/clients of Twine, but rather maintainers and contributors, as a reference and aid when writing docstrings.

Reply:
That makes more sense. I'd like to collaborate on a better name for this section of the docs then. We could nestle this into an internal folder like `flake8` does: https://github.com/PyCQA/flake8/tree/master/docs/source/internal or we could keep these just inside the `internal` folder without an API sub-folder. I'm okay with either or an entirely different name that makes this painfully clear that it's not for users